### PR TITLE
Added sound effects. Issue #2

### PR DESCRIPTION
**Pull request by cosminlupu**

### CODE REVIEW

File: src/components/SoundToggle.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <p>Sound: <a href="#" v-on:click="onClick"><span v-if="sound">On</span><span v-else>Off</span></a></p>
```
Comment (samiheikki) [inline src/components/SoundToggle.vue]: Can this have sound off/on icon instead? Like: https://material.io/icons/#ic_volume_off https://material.io/icons/#ic_volume_up

File: src/App.vue [hunk 2]
```diff
@@ -65,6 +71,24 @@ export default {
...
+    let localStorageSound = window.localStorage.getItem('sound')
+    this.sound = localStorageSound ? JSON.parse(localStorageSound) : true
+
+    this.gameoverSound = new Howl({
+      src: ['../static/sounds/gameover.mp3', '../static/sounds/gameover.ogg'],
+      rate: 1.3,
+      volume: 0.5
+    })
+    this.correctSound = new Howl({
+      src: ['../static/sounds/correct.mp3', '../static/sounds/correct.ogg'],
```
Comment (samiheikki) [inline src/App.vue]: Correct sound was the only one that I did not like. Can this be changed to something a little less disturbing "correct" sound?

File: src/App.vue [hunk 3]
```diff
@@ -65,6 +71,24 @@ export default {
       this.updateLogo()
       this.updateOptions()
     })
+
+    // Sound default on
+    let localStorageSound = window.localStorage.getItem('sound')
```
Comment (samiheikki) [inline src/App.vue]: Can you move these to a `initializeSounds` etc method, so `created` method would not become more complex

**Review by samiheikki — COMMENTED** [review]
Super thanks to you @cosminlupu! I was really impressed 🎉 I added a couple of comments in there. Please let me know if you want to continue. If not, I can continue the work from this branch. Again, thank you!

**Comment by cosminlupu** [comment]
Hi @samiheikki,

I've added the required improvements. Regarding the "correct" sound, if the new one i added is not ok, please take over and replace the mp3 and ogg files with some that you like. Another option might be to adjust the `rate` option in the sound initialisation to change it a bit. You can see more about that on the [Howler Repo](https://github.com/goldfire/howler.js#rate-number-10).

**Comment by samiheikki** [comment]
Thank you @cosminlupu !!!!!! 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**